extension/chrome/elements/compose-modules/compose-storage-module.ts

```diff
@@ -75,28 +75,39 @@ export class ComposeStorageModule extends ViewModule<ComposeView> {
 
   public lookupPubkeyFromKeyserversThenOptionallyFetchExpiredByFingerprintAndUpsertDb = async (
     email: string, name: string | undefined
-  ): Promise<Contact | "fail"> => {
+  ): Promise<Contact[] | "fail"> => {
```

Comment:
Hi @IvanPizhenko 
`Contact` isn't stored in the database
- `test/source/platform/store/contact-store.ts` -- this is a mock database used only in tests, so it is implemented to the extent required for tests to run -- it doesn't fully mimick the real store's behaviour.
- `extension/js/common/platform/store/contact-store.ts` -- this is the real one.
As you can see, only `Pubkey`, `Email` and `Revocation` entities are stored there. `Contact` type is only left for some consumer classes that we haven't refactored yet -- it emulates the older storage by picking the best candidate pubkey for emal and including it in the `Contact`. If consumer class needs more data then the best pubkey, it shouldn't use `Contact` type nor `ContactStore.get`. We will remove these type and method when no more consumers are left (updating the mock `contact-store.ts` in due course).